tests/tests.js

```diff
@@ -822,7 +822,7 @@ const namedGroupFixtures = [
 	{
 		'pattern': '\\k<name>\\k<name>(?<name>)\\k<name>',
 		'flags': '',
-		'expected': '\\1\\1()\\1'
+		'expected': '(?:)(?:)()\\1'
```

Comment:
This output is equivalent, right? The reason for this change is that now some named references may need to be replaced with multiple indexed references, so just keeping a (js) ref to the AST node for `\\k<name>` and mutating it does not work anymore.

Reply:
Ack